platform/windows/os_windows.cpp

```diff
@@ -156,6 +156,52 @@ void RedirectIOToConsole() {
 	}
 }
 
+bool OS_Windows::is_using_con_wrapper() const {
+	static String exe_renames[] = {
+		".console.exe",
+		"_console.exe",
+		" console.exe",
+		"console.exe",
+		String(),
+	};
```

Comment:
Should we also support `-console.exe`?

Reply:
It's not supported by wrapper, be I guess could be added.
https://github.com/godotengine/godot/blob/b00e1cbf743dcb6a2b7f6bd14e348a1a7cf3d403/platform/windows/console_wrapper_windows.cpp#L74-L80